scripts/jenkins/jenkinsfiles/Jenkinsfile-Master

```diff
@@ -41,6 +41,7 @@ try {
   }
 } finally {
   if (pipelineContext != null) {
-    pipelineContext.getEmailer().sendEmail(this, result, pipelineContext.getBuildSummary().getSummaryHTML(this))
+// FIXME uncomment
```

Comment:
Because I needed to test it on c1 (just to make sure). And I don't want the pipeline to fire a notification. Maybe I could add a parameter which will disable emails for branches other than master and rel-*